chart/validator-plugin-oci/values.yaml

```diff
@@ -51,8 +51,12 @@ metricsService:
     targetPort: https
   type: ClusterIP
 auth:
-  secretName: aws-creds
-  createSecret: true
-  accessKeyId: ""
-  secretAccessKey: ""
-  sessionToken: ""
+  secrets:
+    # Specify the names of secrets in your cluster that contain a registries credentails.
+    # E.g.: https://github.com/spectrocloud-labs/validator/blob/main/chart/validator/templates/plugin-secret-oci.yaml
+    #- secretName: oci-registry-creds
+    #- secretName: ecr-registry-creds
```

Comment:
wanted to get thoughts on the way this is setup. auth secrets for the oci plugin are a little different than the other plugins because a user could be validating against an arbitrary number of registries which could each require their own set of auth creds.

let me know if there's a better approach to handling multiple creds. another option would be to only support 1 secret, but to me that doesn't feel like the right thing to do.

also as an aside, we're going to need to create the `plugin-secret-oci.yaml` file in the validator repo. it doesn't exist at this point.